pkg/client/mapiserver/snapshot.go

```diff
@@ -70,18 +69,9 @@ func CreateSnapshot(volName string, snapName string) error {
 	}
 	defer resp.Body.Close()
 
-	body, err := ioutil.ReadAll(resp.Body)
```

Comment:
We have to read this body here , to get the proper error  which is return as a body